internal/services/compute/proximity_placement_group_resource.go

```diff
@@ -47,6 +50,24 @@ func resourceProximityPlacementGroup() *pluginsdk.Resource {
 
 			"location": commonschema.Location(),
 
+			"intent_vm_sizes": {
+				Type:     pluginsdk.TypeSet,
+				Optional: true,
+				MinItems: 1,
+				Elem: &pluginsdk.Schema{
+					Type:         pluginsdk.TypeString,
+					ValidateFunc: validation.StringIsNotWhiteSpace,
```

Comment:
the Compute SDK exposes a list of VM SKU's?

Reply:
have updated to `virtualmachines.PossibleValuesForVirtualMachineSizeTypes()`